src/Symfony/Component/Security/Core/SecurityContext.php

```diff
@@ -60,11 +60,7 @@ public function __construct(AuthenticationManagerInterface $authenticationManage
             $this->token = $this->authenticationManager->authenticate($this->token);
         }
 
-        if (!is_array($attributes)) {
-            $attributes = array($attributes);
-        }
-
-        return $this->accessDecisionManager->decide($this->token, $attributes, $object);
+        return $this->accessDecisionManager->decide($this->token, (array) $attributes, $object);
```

Comment:
This is a BC break, because you are handling Traversable objects in a different way now.

We explicitly stopped using casting here to fix a bug a while ago: https://github.com/symfony/symfony/issues/2794